lib/rbi/index.rb

```diff
@@ -175,6 +175,16 @@ def index_ids
     end
   end
 
+  class TypeMember
```

Comment:
Yeah, I don't think we need to make everything `Indexable`, but it is also very error prone to not have a warning that a merge might remove your newly defined node types from the tree, because it is not one of the types that's handled explicitly.